website/components/LandingPage/index.tsx

```diff
@@ -326,6 +336,7 @@ export default function LandingPage(): ReactElement {
           <video autoPlay={allowMotion} loop muted>
             <source src={BannerVideo} type="video/mp4" />
           </video>
+          <div></div>
```

Comment:
Just curious: why is this empty `<div>`?

Reply:
This div is manipulated by lines 58-64 to have the gradient fill!